kolibri/utils/options.py

```diff
@@ -271,6 +271,26 @@ def multiprocess_bool(value):
         return False
 
 
+def storage_option(value, *opts):
+    """
+    Validate the storage options.
+    Check that the given option is valid, then check that needed external
+    libraries are available where relevant.
+    """
+    value = is_option(value, *opts)
+    if value == "gcs":
+        try:
+            from storages.backends.gcloud import GoogleCloudStorage  # noqa
+
```

Comment:
So my thought was we could try to initialize the credentials here - presumably it throws a predictable error which we can catch and raise the validation error.

Reply:
Looks like we've not completely resolved this?